### PR TITLE
Sliding expiration

### PR DESCRIPTION
**Pull request by Vitalii-Misechko**
Added method `setFileExpiration` which internally use `s3.copyObject` as a well-known workaround. Fixed implementation of `putFile`

### CODE REVIEW

**Comment by Vitalii-Misechko** [comment]
[Relate PR](https://github.com/Brightspace/document-builder/pull/52)

**Comment by Vitalii-Misechko** [comment]
In progress. Writing tests

**Comment by Vitalii-Misechko** [comment]
Unfortunately `fake-s3` doesn't update `last modified` date for `copyObject` function call. Thus I didn't find a way to write tests for it. As a result, coverage decreased.

**Comment by Vitalii-Misechko** [comment]
PRs:
- [Main PR](https://github.com/Brightspace/document-builder-common/pull/23)
- https://github.com/Brightspace/document-builder-worker/pull/21
- https://github.com/Brightspace/document-builder/pull/52

**Comment by Vitalii-Misechko** [comment]
In progress

**Comment by Vitalii-Misechko** [comment]
Ready for review

File: src/util/file-storage.js [hunk 1]
```diff
@@ -42,22 +42,32 @@ class FileStorage {
...
 					Body: stream,
-					Expires: expirationTime,
 					ContentType: contentType
 				});
 			});
 	}
 
+	/**
+	 * Resets file expiration. It changes LastModified file property.
+	 * Please PAY ATTENTION, it clears all metadata values, except ContentType
```
Comment (Vitalii-Misechko) [inline src/util/file-storage.js]: For DCS. this line is not an issue

**Comment by D2LChrisC** [comment]
Ok, looks good!

**Pull request merged**